bower.json

```diff
@@ -47,8 +47,6 @@
     "patternfly-timeline": "~1.0.3",
     "phantomjs-polyfill": "~0.0.2",
     "qs": "~0.3.10",
-    "rx-angular": "rx.angular#~1.1.3",
```

Comment:
Well, I'm only really removing it because it is *never* used..

But it also seems like ui-components is not using Rx *at all*, so we can just remove the two dependencies..